code/modules/clothing/head/helmet.dm

```diff
@@ -372,14 +372,16 @@
 	strip_delay = 80
 
 /obj/item/clothing/head/helmet/elder_atmosian
-	name = "Elder Atmosian Helmet"
+	name = "\improper Elder Atmosian Helmet"
 	desc = "A superb helmet made with the toughest and rarest materials available to man."
-	icon_state = "knight_greyscale"
-	inhand_icon_state = "knight_greyscale"
-	armor = list(MELEE = 15, BULLET = 10, LASER = 30, ENERGY = 30, BOMB = 10, BIO = 10, RAD = 20, FIRE = 65, ACID = 40, WOUND = 15)
-	material_flags = MATERIAL_COLOR | MATERIAL_AFFECT_STATISTICS //Can change color and add prefix
+	icon_state = "h2helmet"
+	inhand_icon_state = "h2helmet"
+	armor = list(MELEE = 40, BULLET = 20, LASER = 30, ENERGY = 30, BOMB = 20, BIO = 10, RAD = 40, FIRE = 100, ACID = 40, WOUND = 30)
+	custom_materials = list(/datum/material/metalhydrogen = 5)
```

Comment:
5??

Reply:
5 and 8 are the amounts of material used to make each armor piece

Reply:
a sheet is 2000 units of material

Reply:
my brain is not massive